test/functional/test_framework/address.py

```diff
@@ -193,7 +193,10 @@ def address_to_scriptpubkey(address):
         return keyhash_to_p2pkh_script(payload)
     elif version == 196:  # testnet script hash
         return scripthash_to_p2sh_script(payload)
-    # TODO: also support other address formats
```

Comment:
nit: There are other address types to support so the todo comment should not be removed